modules/@apostrophecms/ui/ui/apos/components/AposCombo.vue

```diff
@@ -88,26 +104,51 @@ export default {
     value: {
       type: Object,
       required: true
+    },
+    typehead: {
+      type: Boolean,
+      default: false
+    },
+    busy: {
+      type: Boolean,
+      default: false
+    },
+    // When true, indicates that the values and selected items are arrays of objects,
+    // array of primitive values otherwise.
+    objectValues: {
+      type: Boolean,
+      default: false
     }
   },
 
-  emits: [ 'select-items' ],
+  emits: [ 'select-items', 'toggle', 'search' ],
```

Comment:
[Optional] I would have called the `search` one `input` to be more generic but it's certainly a detail.

Reply:
I agree